Maps/ArchipelagoCampaign/WoL/ap_welcome_to_the_jungle.SC2Map/MapScript.galaxy

```diff
@@ -89,6 +107,17 @@ unit gv_victoryTerrazine;
 bool gv_victoryCinematicCompleted;
 
 void InitGlobals () {
+    gv_location_ids.lv_cLOSE_RELIC = 1;
+    gv_location_ids.lv_wEST_RELIC = 2;
+    gv_location_ids.lv_nORTH_EAST_RELIC = 3;
+    gv_location_ids.lv_mIDDLE_BASE = 4;
+    gv_location_ids.lv_mAIN_BASE = 5;
+    gv_location_ids.lv_nO_TERRAZINE_NODES_SEALED = 6;
+    gv_location_ids.lv_uP_TO_1_TERRAZINE_NODES_SEALED = 7;
+    gv_location_ids.lv_uP_TO_2_TERRAZINE_NODES_SEALED = 8;
+    gv_location_ids.lv_uP_TO_3_TERRAZINE_NODES_SEALED = 9;
+    gv_location_ids.lv_uP_TO_4_TERRAZINE_NODES_SEALED = 10;
+    gv_location_ids.lv_uP_TO_5_TERRAZINE_NODES_SEALED = 11;

```

Comment:
Curious if this initialization requires you to define the structure of `location_ids` separately from the default global values. If it _does_ require you to work in two places, I'd recommend specifying them all as raw global constants, e.g. `LOCATION_ID_UP_TO_5_TERRAZINE_NODES_SEALED`. Having a standard prefix should make search just as easy. If everything is specified in one place, then it's not an issue and you should keep going this route / the one you prefer.

Reply:
That's a valid point. I originally envisioned using a 'record' to make the GUI allow users to use a drop-down of options, but looking at it again I agree with your stand. I will replace these with basic constants. Thank you for the idea!